src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs

```diff
@@ -59,6 +60,10 @@ public static void ConfigureMvcOptions(MvcOptions options)
             options.ValueProviderFactories.Add(new QueryStringValueProviderFactory());
             options.ValueProviderFactories.Add(new FormValueProviderFactory());
 
+            // Set up metadata providers
+            options.ModelMetadataProviders.Add(new DefaultModelMetadataBindingDetailsProvider());
+            options.ModelMetadataProviders.Add(new DataAnnotationsModelMetadataDetailsProvider());
```

Comment:
no default `IModelMetadataValidationDetailsProvider` i.e. no validation provider to add here?